go/coordinator/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/chroma/chroma-coordinator
 go 1.20
 
 require (
+	github.com/go-gormigrate/gormigrate/v2 v2.1.1
```

Comment:
For posterity's sake can you elaborate on why this choice of migration tool - what the alternatives considered were?

I think this is a hard to switch thing.